### PR TITLE
Upgrade to mutant-0.10.19

### PR DESCRIPTION
**Pull request by mbj**
* This release changes integration packaging to explicitly stick to its
  mutant main gem version. This avoids errors when only mutant or
  mutant-minitest / mutant-rspec gets upgraded.

### CODE REVIEW

**Comment by mbj** [comment]
I could not upgrade depencencies in `contrib/ruby_event_store-outbox` due:

```
mutant-dev@mbj ~/devel/rails_event_store/contrib/ruby_event_store-outbox (upgrade/mutant*) $ bundle install --gemfile Gemfile.rails_5_1
Fetching gem metadata from https://gem.mutant.dev/.
Fetching gem metadata from https://gem.mutant.dev/.
Fetching gem metadata from https://rubygems.org/..........
Fetching gem metadata from https://gem.mutant.dev/.
Fetching gem metadata from https://gem.mutant.dev/.
Fetching gem metadata from https://rubygems.org/..
Resolving dependencies...
Bundler could not find compatible versions for gem "activerecord":
  In snapshot (Gemfile.rails_5_1.lock):
    activerecord (= 5.1.7)

  In Gemfile.rails_5_1:
    rails (= 5.1.7) was resolved to 5.1.7, which depends on
      activerecord (= 5.1.7)

    ruby_event_store-outbox was resolved to 0.0.15, which depends on
      activerecord (>= 5.2)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

`bundle update` ran into the same dependency violation. The policies on version support, in this project, are unknown to me hence I've to stop at this point.

File: bounded_context/Gemfile.rails_6_0.lock [hunk 1]
```diff
@@ -0,0 +1,218 @@
+PATH
```
Comment (mbj) [inline bounded_context/Gemfile.rails_6_0.lock:1]: @pawelpacana There was no corresponding lockfile, so I added it.

File: rails_event_store/Gemfile.rails_6_0.lock [hunk 2]
```diff
@@ -0,0 +1,295 @@
+PATH
```
Comment (mbj) [inline rails_event_store/Gemfile.rails_6_0.lock:1]: Instance of: https://github.com/RailsEventStore/rails_event_store/pull/905/files#r542507198

File: rails_event_store_active_record/Gemfile.rails_6_0.lock [hunk 3]
```diff
@@ -0,0 +1,241 @@
+PATH
```
Comment (mbj) [inline rails_event_store_active_record/Gemfile.rails_6_0.lock:1]: Instance of: https://github.com/RailsEventStore/rails_event_store/pull/905/files#r542507198

**Comment by mbj** [comment]
@pawelpacana Ready for your review. I PRed this update manually as the new constraint from `mutant-minitest` to `mutant` was something I wanted to see on a more complex project. Success IMO.

File: bounded_context/Gemfile.rails_6_0.lock (hunk 1 above)
Comment (pawelpacana) [reply bounded_context/Gemfile.rails_6_0.lock:1]: 👍 noticed it missing as well, recent addition not caught on CI (#904)

Comment (pawelpacana) [reply bounded_context/Gemfile.rails_6_0.lock:1]: 👍 noticed it missing as well, recent addition not caught on CI 

> I could not upgrade depencencies in `contrib/ruby_event_store-outbox` due:
> 
> ```
> mutant-dev@mbj ~/devel/rails_event_store/contrib/ruby_event_store-outbox (upgrade/mutant*) $ bundle install --gemfile Gemfile.rails_5_1
> Fetching gem metadata from https://gem.mutant.dev/.
> Fetching gem metadata from https://gem.mutant.dev/.
> Fetching gem metadata from https://rubygems.org/..........
> Fetching gem metadata from https://gem.mutant.dev/.
> Fetching gem metadata from https://gem.mutant.dev/.
> Fetching gem metadata from https://rubygems.org/..
> Resolving dependencies...
> Bundler could not find compatible versions for gem "activerecord":
>   In snapshot (Gemfile.rails_5_1.lock):
>     activerecord (= 5.1.7)
> 
>   In Gemfile.rails_5_1:
>     rails (= 5.1.7) was resolved to 5.1.7, which depends on
>       activerecord (= 5.1.7)
> 
>     ruby_event_store-outbox was resolved to 0.0.15, which depends on
>       activerecord (>= 5.2)
> 
> Running `bundle update` will rebuild your snapshot from scratch, using only
> the gems in your Gemfile, which may resolve the conflict.
> ```
> 
> `bundle update` ran into the same dependency violation. The policies on version support, in this project, are unknown to me hence I've to stop at this point.

cc @swistak35

**Comment by pawelpacana** [comment]
Just in time for scheduled run: https://github.com/RailsEventStore/rails_event_store/runs/1551849791?check_suite_focus=true

**Comment by pawelpacana** [comment]
@mbj without knowing anything about the load on test machine:

1. yesterday on 0.10.17, completed / https://github.com/RailsEventStore/rails_event_store/runs/1546238884?check_suite_focus=true

        progress: 11564/11565 alive: 0 runtime: 5027.37s killtime: 1453.64s mutations/s: 2.30

2. today on 0.10.19, timed out / https://github.com/RailsEventStore/rails_event_store/runs/1551850214?check_suite_focus=true

        progress: 9327/11565 alive: 0 runtime: 7191.94s killtime: 2350.12s mutations/s: 1.30

**Comment by mbj** [comment]
@pawelpacana At these large subject amounts I'd say this is all variance at 2 samples. I know for a fact .17 and .19 had only marginal changes (but a selector fix that likely does not affect you as you use no custom operator implementations).

Very likely noise related. I've got more improvements for longer runs in the pipeline:

* Avoid "VM trashing" throug better main process architecture.
* Allow native coverage pass splitting to distribute large passes into smaller ones.

**Comment by mbj** [comment]
So lets wait a couple of full runs till taking action.

**Comment by pawelpacana** [comment]
Added manual trigger, adjusting timeout to be more generous.

Zrzut ekranu 2020-12-14 o 20 24 40

File: bounded_context/Gemfile.rails_6_0.lock (hunk 1 above)
Comment (pawelpacana) [reply bounded_context/Gemfile.rails_6_0.lock:1]: found reasoning in bb5257c1464cb253f84825ed66b25d877e2f5e68 and removed extra Gemfiles

**Pull request merged**